WcaOnRails/app/webpacker/components/RolesTab/RoleForm.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import { Form } from 'semantic-ui-react';
+
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import { roleDataUrl, roleUpdateUrl } from '../../lib/requests/routes.js.erb';
+import useSaveAction from '../../lib/hooks/useSaveAction';
+import Errored from '../Requests/Errored';
+import DelegateForm from './DelegateForm';
+
+const groups = [{
+  key: 'delegates',
+  text: 'Delegates',
+  value: 'Delegates',
+}];
+
+const endRoleValue = {
+  delegateStatus: '',
+  seniorDelegateId: '',
+  location: '',
+};
+
+export default function RoleForm() {
+  const userId = window.location.pathname.split('/')[2];
+  const roleId = window.location.pathname.split('/')[4];
+  const { data, loading, error } = useLoadedData(roleDataUrl(userId, roleId));
+  const { save } = useSaveAction();
+  const [selectedGroup] = React.useState(groups[0].value);
```

Comment:
Why do we need a state if it never changes?

Reply:
Changed it to `const selectedGroup = groups[0].value;`. It is actually supposed to be a state, but currently the `groups` array have only one value, so there is no case where it will be changed. In future, when more groups come, I'll bring back the `useState`.